Automattic/wp-calypso
# Framework: Remove (almost all of) tinymce's usage 🎉

#### Changes proposed in this Pull Request

After the recent deprecation and removal of the WooCommerce store in Calypso, the `CompactTinyMCE` component is no longer in use. When we remove it, tinymce is no longer in use in Calypso, too - with the exception of the `wpcom-block-editor` app.

This PR cleans up the obsolete tinymce usage, including:

* Removing `CompactTinyMCE`
* Moving the old `wpcom-track-paste` sources to a Calypso client lib, because they're still used in Guided Tours
* Removing the tinymce components directory, which contained old and unused tinymce plugins
* Removing unused tinymce skins
* Removing the server handling of static files that was used solely for the tinymce skins
* Removing special handling of by webpack for tinymce's globals
* Removing tinymce as a dependency of Calypso's client package.

There are more unused WooCommerce components, we're separately removing them in #51012.

#### Testing instructions

* Verify nothing we're removing is used.
* Verify build still works well.
* Verify tests still pass.
